### ISSUE TITLE
Repeating notifications for already available media

### ISSUE THREAD

**Issue by NekoPavel** [Fallenbagel/jellyseerr#466]
### Description

Some shows keep sending out "Media available" notifications

### Version

1.6.0

### Steps to Reproduce

1. Run Sonarr Scan
2. Get notifications about shows that have already sent out notifications

### Logs

```shell
Did not fit here.
https://privatebin.pavel.gg/?e875bfc935dfde5a#FNL9GfiCgcnGjWjj4uWScYQuU9CjeuMZVYShyeDfxFDt
```

### Platform

desktop

### Device

Any

### Operating System

Docker on Ubuntu

### Browser

Any

### Additional Context

Most likely caused by #465 

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by Fallenbagel** [comment]
This is has possibly been fixed in the develop. Can you test out with the develop tag?

**Comment by NekoPavel** [comment]
Yeah seems to be solved by #444 
I assume that the underlying issue for episode number mismatch us caused by #465 however.

**Comment by Fallenbagel** [comment]
Episode mismatch is caused by the difference in database yes but the notification issue is completely unrelated is what I meant.

**Comment by NekoPavel** [comment]
Alright, can be closed when #444 is merged into main
